Gopkg.toml

```diff
@@ -38,32 +37,26 @@ required = [
 
 [[override]]
   name = "sigs.k8s.io/controller-runtime"
-  version = "v0.1.3"
+  version = "v0.1.4"
+
+[[constraint]]
+  name = "github.com/operator-framework/operator-sdk"
+  # The version rule is used for a specific release and the master branch for in between releases.
+  # branch = "master" #osdk_branch_annotation
+  version = "=v0.1.0" #osdk_version_annotation
 
 [prune]
   go-tests = true
   non-go = true
-  unused-packages = true
-
+  
   [[prune.project]]
     name = "k8s.io/code-generator"
     non-go = false
-    unused-packages = false
-
-[[constraint]]
-  name = "github.com/operator-framework/operator-sdk"
-  # The version rule is used for a specific release and the master branch for in between releases.
-  # branch = "master" #osdk_branch_annotation
-  version = "=v0.0.7" #osdk_version_annotation
-
-# Everything above this is required by the SDK. For each SDK version upgrade, the
-# above contents should be replaced by code copied from the SDK repo located at:
-# https://github.com/operator-framework/operator-sdk/blob/8f0fe6975128d3670e92e0509c645d80657a6d68/Gopkg.toml#L9-L23
```

Comment:
@aravindhp I created the toml file by using the operator-sdk to generate the files in the repo, then copying any dependencies that we added manually. The latest sdk doesn't add that comment itself, so my assumption is that it is no longer accurate.

Reply:
The SDK didn't add that comment but I did :-) That comment will be accurate until operator-framework/operator-sdk#638 is solved and should stay until then.

Reply:
Ah, I understand. In this case the dependencies were generated because the file was recreated by the sdk (and I confirmed that by looking at the latest versions of the dependencies in the SDK repo). But I restored your comment for upgrades that don't involve a full recreation.